app/connectors/connector-verifier/src/main/java/io/syndesis/verifier/api/SyndesisMetadata.java

```diff
@@ -33,8 +38,7 @@
      */
     public final Map<String, List<PropertyPair>> properties;
 
-    public SyndesisMetadata(final Map<String, List<PropertyPair>> properties, final T inputSchema,
-                            final T outputSchema) {
+    public SyndesisMetadata(final Map<String, List<PropertyPair>> properties, final T inputSchema, final T outputSchema) {
```

Comment:
can we add a warning comment to the member vars for a heads up that when new meta data is added that the converter class needs to be updated, too ?

Reply:
Warnings are cheap, tests are better, added test to check for that.